cmd/run.go

```diff
@@ -97,6 +97,8 @@ func bindRunFlags(cmd *cobra.Command, config *app.Config) {
 	cmd.Flags().StringVar(&config.Nickname, "nickname", "", "Human friendly peer nickname. Maximum 32 characters.")
 	cmd.Flags().StringSliceVar(&config.BeaconNodeHeaders, "beacon-node-headers", nil, "Comma separated list of headers formatted as header=value")
 	cmd.Flags().StringSliceVar(&config.FallbackBeaconNodeAddrs, "fallback-beacon-node-endpoints", nil, "A list of beacon nodes to use if the primary list are offline or unhealthy.")
+	cmd.Flags().StringSliceVar(&config.Graffiti, "graffiti", nil, "Comma separated list of graffiti strings to include in block proposals. Maximum 32 ASCII characters per graffiti.")
+	cmd.Flags().BoolVar(&config.GraffitiDisableClientAppend, "graffiti-disable-client-append", false, "Disables appending the charon name to the client graffiti.")
```

Comment:
Maybe Oisin knows better, but I would explain in description that it appends "OB" suffix and this option disables this behavior. To be explicit. Otherwise it's unclear what "client" is being appended..